src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -437,64 +452,107 @@ export default {
 				'Europe'
 			],
 			activities: null,
+			enableChallengeHeader: false,
+			teamData: {},
+			shareLender: undefined,
 		};
 	},
 	mixins: [fiveDollarsTest, guestComment, hugeLendAmount],
 	apollo: {
 		query: preFetchQuery,
 		preFetch(config, client, { route, cookieStore }) {
 			const publicId = getPublicId(route);
-			return client
-				.query({
-					query: preFetchQuery,
-					variables: {
-						loanId: Number(route.params?.id ?? 0),
-						publicId,
-						getInviter: !!publicId,
-						basketId: cookieStore.get('kvbskt')
-					},
-				})
-				.then(({ data }) => {
-					const expCookieSignifier = cookieStore.get('kvlendborrowerbeta');
-					if (expCookieSignifier === 'a' || expCookieSignifier === 'c') {
-						const { query } = route;
-						return Promise.reject({
-							path: `/lend-classic/${route.params.id}`,
-							query,
-						});
-					}
+			return Promise.all([
```

Comment:
`Promise.all` not needed here anymore. Reverting that might make this diff slightly easier to read.